torch/csrc/jit/codegen/cuda/scheduler.cpp

```diff
@@ -15,7 +15,7 @@ namespace jit {
 namespace fuser {
 namespace cuda {
 
-constexpr int kUnrollFactor = 4;
+constexpr int kUnrollFactor = 1;
```

Comment:
Same here, if we are removing unrolling code, we can do it cleaner and getting rid of the macro.

Reply:
We will want to re-enable this sooner than later.